src/service/world-api-adapter.js

```diff
@@ -223,6 +219,24 @@ module.exports = function (config) {
         },
 
         /**
+        * Set the filter for the current instance of the world adapter
+        *
+        * **Example**
+        * var ws = new F.service.World({...}).load('123').addUser({ userId: '123' });
+        *
+        *
+        */
+        load: function (worldId) {
+            if (!worldId || typeof worldId !== 'string') {
+                throw new Error('load needs a worldId string to load (' + worldId + ')');
```

Comment:
but it'll only get into the error message if `typeof worldId !== 'string'` right?